File: ciclos.py
# Ciclos
""" 
for in
while

"""
numeros = [1, 2, 3, 4]

# imprimir los numeros del 1 al 10
for i in range(1, 11): # range(start=0, stop=10, step=1)
    print(i)
    
    
for i in range(len(numeros)):
    print(numeros[i]) # 1 2 3
    
for numero in numeros:
    print(numero)


indice = 0
while indice < len(numeros):
    print(numeros[indice])
    indice+=1
    #indice = indice + 1

File: ciclos.js
/* Ciclos */
/* 

for

for in

for of

while

do-while

*/

let numeros = [1, 2, 3]

for(let i = 1; i <= 10; i++){
    console.log(i)
}

for(let i = 0; i < numeros.length; i++){
    console.log(numeros[i])
}

for(let indice in numeros){
    console.log(numeros[indice]) // 1 2 3
}

for(let valor of numeros){
    console.log(valor) // 1 2 3
}

let indice = 0;
while(indice < numeros.length){
    console.log(numeros[indice])
    indice++
}